### PR TITLE
Add AWESOME.md

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*

*Description of changes:*
Add AWESOME.md

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by yinweisu — APPROVED** [review]
LGTM!

**Review by gradientsky — COMMENTED** [review]
Add this benchmark: [Benchmarking Automated Machine Learning Methods for Price Forecasting Applications](https://arxiv.org/abs/2304.14735)

**Comment by Innixma** [comment]
> Add this benchmark: [Benchmarking Automated Machine Learning Methods for Price Forecasting Applications](https://arxiv.org/abs/2304.14735)

Already added

File: AWESOME.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+5. (Live, 2023/03/11) Top 3% : [Titanic - Machine learning from Disaster](https://www.kaggle.com/competitions/titanic/discussion/394105#2178151)
+6. (Hist,  2022/03/22) Rank 24/3507 : [Otto Group Product Classification Challenge](https://www.kaggle.com/code/innixma/autogluon-rank-24-in-1-line-of-code)
+
+## AutoML Benchmarks using AutoGluon
+* [AMLB: An AutoML Benchmark](https://openml.github.io/automlbenchmark/)
+  * For a thorough comparison of AutoGluon and other modern AutoML systems, please refer to the 2022 paper ["AMLB: An AutoML Benchmark"](https://arxiv.org/abs/2207.12560) where AutoGluon is shown to be the state-of-the-art among AutoML systems on tabular data.
+  * We encourage all users to benchmark AutoGluon & other AutoML frameworks on AMLB.
+  * This is our preferred benchmark as it is widely accepted and trusted within the AutoML community.
+
+## Papers using AutoGluon
```
Comment (tonyhoo) [inline AWESOME.md:33]: Shall we have a section for *Tools with AutoGluon* which can include cloud and benchmark? Also community would be helpful as well (e.g. Discord, Slack)

**Pull request merged**